frontend/lib/norent/homepage.tsx

```diff
@@ -294,33 +319,38 @@ export const NorentHomePage: React.FC<{}> = () => {
               rather than a detailed example of what an actual letter looks like. */}
               <article className="message">
                 <div className="message-body has-background-grey-lighter has-text-left has-text-weight-light">
-                  <p>Dear Landlord/Management.</p>
-                  <br />
-                  <p>
-                    I am writing to inform you that I have experienced a loss of
-                    income, increased expenses and/or other financial
-                    circumstances related to the pandemic. Until further notice,
-                    the COVID-19 emergency may impact my ability to pay rent.
-                  </p>
-                  <br />
                   <p>
-                    Tenants in Florida are protected from eviction for
-                    non-payment by Executive Order 20-94, issued by Governor Ron
-                    DeSantis on April 2, 2020.
+                    <Trans>Dear Landlord/Management.</Trans>
                   </p>
                   <br />
-                  <p className="is-hidden-mobile">
-                    Tenants in covered properties are also protected from
-                    eviction, fees, penalties, and other charges related to
-                    non-payment by the CARES Act (Title IV, Sec. 4024) enacted
-                    by Congress on March 27, 2020.
-                  </p>
-                  <br />
-                  <p className="is-hidden-mobile">
-                    Along with my neighbors, I am organizing, encouraging,
-                    and/or participating in a tenant organization so that we may
-                    support
-                  </p>
+                  <Trans>
+                    <p>
+                      I am writing to inform you that I have experienced a loss
+                      of income, increased expenses and/or other financial
+                      circumstances related to the pandemic. Until further
+                      notice, the COVID-19 emergency may impact my ability to
+                      pay rent.
+                    </p>
+                    <br />
```

Comment:
These `<br/>`s inside `<Trans>` content concern me a bit because they're essentially formatting/styling rather than content, and they could confuse translators (especially since they can't see the actual type of tag and will just see e.g. `<1>`... and  if we ever remove them, it will cause translators to have to re-translate the text, which could be frustrating.  It'd be nice to get rid of them but I also don't want this PR to have to also tackle CSS stuff, so maybe file an issue to address it later?

Reply:
Hey hey, any thoughts on this?  I think it's the only part of my review that you haven't addressed but I might have missed something.

Reply:
Oooh sorry! I filed an issue for this as you suggested (https://github.com/JustFixNYC/tenants2/issues/1424) but forgot to circle back and link it here.